src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -213,6 +213,14 @@
  *   String firstURL; // OK, URL abbreviation is allowed
  *   final int TOTAL = 5; // OK, final is ignored
  *   static final int LIMIT = 10; // OK, static final is ignored
+ *   void newOAuth2Client() {} // violation, split occurs as 'new', 'O', 'Auth2', 'Client'
+ *                                    // and not as 'new', 'OA', ''uth2', 'Client'.
+ *                                    // beginIndex = 3, endIndex = 4,
+ *                                    // The char at endIndex is included in the abbreviation if
+ *                                    //  it is the last character in the string.
+ *   void ClientOA() {} // OK, Abbreviation is at end
+ *   void OAClient() {} // OK, split occurs as 'OA', 'Client'
+ *
```

Comment:
Please use OAuth everywhere.
Users do care about real examples.
Please show what config should be to not trigger violation on any usage of OAuth in names

Reply:
done